cockpit/webpack.config.js

```diff
@@ -233,8 +217,98 @@ module.exports = {
                 loader: 'file-loader',
             },
             {
-                test: /\.less$/,
-                loader: ExtractTextPlugin.extract("css-loader?sourceMap&minimize=!less-loader?sourceMap&compress=false")
```

Comment:
It looks like a big change. Could you please add more comments to this code?

Reply:
Sure I will add some more comments.